src/dos/drive_local.cpp

```diff
@@ -774,23 +778,22 @@ cdromDrive::cdromDrive(const char _driveLetter,
 	if (MSCDEX_GetVolumeName(subUnit,name)) dirCache.SetLabel(name,true,true);
 }
 
-bool cdromDrive::FileOpen(DOS_File** file, const char* name, uint8_t flags)
+std::unique_ptr<DOS_File> cdromDrive::FileOpen(const char* name, uint8_t flags)
 {
 	if ((flags & 0xf) == OPEN_READWRITE) {
 		flags &= ~static_cast<unsigned>(OPEN_READWRITE);
 	} else if ((flags & 0xf) == OPEN_WRITE) {
 		DOS_SetError(DOSERR_ACCESS_DENIED);
-		return false;
+		return nullptr;
 	}
-	bool success = localDrive::FileOpen(file, name, flags);
-	return success;
+	return localDrive::FileOpen(name, flags);
 }
 
-bool cdromDrive::FileCreate(DOS_File** /*file*/, const char* /*name*/,
-                            FatAttributeFlags /*attributes*/)
+std::unique_ptr<DOS_File> cdromDrive::FileCreate(const char* /*name*/,
```

Comment:
How about `[[maybe_unused]]` instead of commenting-out param?

Reply:
Sure.  Changed.